dll/win32/shell32/wine/shell32_main.c

```diff
@@ -1135,6 +1135,47 @@ INT_PTR CALLBACK AboutAuthorsDlgProc( HWND hWnd, UINT msg, WPARAM wParam, LPARAM
 /*************************************************************************
  * AboutDlgProc            (internal)
  */
+#ifdef __REACTOS__
+// Check if the version is Workstation by reading the CSDVersion string
+// This will be used to choose the correspondent ReactOS Bitmap banner
+// on the About dialog
+static BOOL IsVersionWorkstation(VOID)
+{
+    DWORD dwType = 0, dwValue = 0, dwSize = 0;
+    HKEY hKey = NULL;
+    LONG lRet = 0;
+
+    lRet = RegOpenKeyExW(HKEY_LOCAL_MACHINE,
+                         L"SYSTEM\\CurrentControlSet\\Control\\Windows",
+                         0,
+                         KEY_QUERY_VALUE,
+                         &hKey);
+    if (lRet != ERROR_SUCCESS)
+        return FALSE;
+
+    // Check product version number
+    dwValue = 0;
```

Comment:
What's the point of assigning `dwValue` to 0 again when it's already assigned before?